runtime/onert/core/src/loader/circle_traininfo.fbs

```diff
@@ -1,4 +1,4 @@
-// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/91fc9550d914a3c62bf41e62e007e1a2f5f37b59/spec/circle_training.fbs
+// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/e94b2b8956bcc316bef19dc4885a5027f545845a/spec/circle_training.fbs
```

Comment:
This is to sync with the inner-source version.

Reply:
(Not for this PR) 
Now, this file follows inner-source spec using commit-id. 
But it might be easy to maintain if we menage `circle_traininfo.fbs` in a public repo like circle schema.